documentation/src/main/markdown/packagemgrs/opam.md

```diff
@@ -0,0 +1,37 @@
+# Opam Support
```

Comment:
This page also needs updating.
https://documentation.blackduck.com/bundle/detect/page/components/detectors.html

Reply:
Yeah just realized that, will push one commit soon.